curriculum/js1/4.js

```diff
@@ -6,8 +6,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (n, b) => {
+  if (n===1) {
+    return b;
+  }
+  return b+solution(n-1, b);
```

Comment:
How could you write this return without this construct (adding `b` and the solution call)?

Reply:
I included the result into the function. Thanks for the review!